[EDGE CIE-5221] y2025 DataHub installation using the Edge Operator

bbyreddy opened the pull request:
- DataHub for Edge on K8s updates
- Restructured and moved the DataHub content for the Edge Appliance VM into the Edge docs, so that the Edge content in the DataHub docs is all related to Edge on K8s.

bbyreddy committed:
datahub docs

bbyreddy committed:
Merge branch 'edge/y2025-updates-first-cut' into edge/y2025-datahub-integration

bbyreddy committed:
Updates

bbyreddy committed:
updates

bbyreddy committed:
Updates to CR and prerequisites

bbyreddy committed:
Merge branch 'edge/y2025-updates-first-cut' into edge/y2025-datahub-integration

bbyreddy committed:
Update prerequisites.md

bbyreddy committed:
Update configuring-storage.md

bbyreddy committed:
Update configuring-storage.md

bbyreddy committed:
updates

bbyreddy committed:
Update configuring-storage.md

bbyreddy committed:
Update configuring-storage.md

bbyreddy committed:
Update configuring-storage.md

bbyreddy committed:
Update configuring-storage.md

bbyreddy committed:
updates

bbyreddy committed:
Update configuring-storage.md

bbyreddy committed:
Update setting-up-datahub-edge-on-k8s.md

bbyreddy committed:
Update k8s-edge-messaging-service.md

rthrippleton commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
```diff
@@ -0,0 +1,128 @@
+
+#### Add entry to /etc/hosts
+
+In order to access Dremio, the following entry needs to be added to ``/etc/hosts``:
```
This is a bit weird - I thought we expected customers to have proper DNS? Or do we do this for other Edge-related hostnames?

bbyreddy replied:
Yeah :( , this is what we also add for the Edge/Management domains in the accessing Edge section. 
https://cumulocity-iot-c8y-docs-build-pr-3131.surge.sh/edge-kubernetes/installing-edge-on-k8/#accessing-cumulocity-iot-edge-using-the-domain-name

May be we can repeat the same in case of DataHub as well. If you have DNS configure them accordingly else update the /etc/hosts file. What do you suggest?

rthrippleton replied:
4aa7475d6

bbyreddy replied:
Thank you, @rthrippleton

rthrippleton committed:
Don't hardcode use of /etc/hosts for datahub-<domain_name>

rthrippleton commented on content/datahub/running-datahub-on-the-edge-bundle/datahub-edge-overview.md:
```diff
@@ -30,19 +29,25 @@ For your convenience, here is an overview of the contents:
+#### {{< product-c8y-iot >}} DataHub Edge variants {#datahub-edge-variants}
+{{< product-c8y-iot >}} DataHub Edge is available in two variants corresponding to two 
+delivery modes of {{< product-c8y-iot >}} Edge:
+* An appliance running in a Linux VM deployed into a supported hypervisor. In this setup, the {{< product-c8y-iot >}} DataHub backend, the Dremio master, and the Dremio executor are running as Docker containers
```
I know our doc versioning is a bit weird, but won't this eventually appear as if it's doc for DataHub 2025?

In which case, we shouldn't talk about the appliance, because we don't support DataHub 2025 on appliance 10.18.

BeateRixen replied:
Right, this is the documentation for DataHub 2025.

bbyreddy replied:
We do support DataHub 2025 on 10.18 Appliance as this combination is continuously being tested by the DataHub team. I will get it confirmed by the DataHub team.

rthrippleton commented on content/datahub/running-datahub-on-the-edge-bundle/datahub-edge-overview.md:
```diff
@@ -30,19 +29,25 @@ For your convenience, here is an overview of the contents:
-| Dremio container management | Docker daemon | Kubernetes |
-| {{< product-c8y-iot >}} DataHub backend container management | Docker daemon | Microservice in {{< product-c8y-iot >}} Core |
-| Data lakes | NAS | Azure Storage, S3, HDFS, (NAS) |
+| Area | {{< product-c8y-iot >}} DataHub Edge | {{< product-c8y-iot >}} DataHub Edge Kubernetes | {{< product-c8y-iot >}} DataHub Cloud |
```
Same here. Shouldn't be comparing appliance and Edge-on-K8S.

bbyreddy replied:
Edge Appliance 10.18 is supported at least until the end of 2025, hence didn't change it. Moreover, the 2025 release doc will still have the Edge Appliance 10.18 tile along side of the 2025 Edge on K8s tile.

rthrippleton commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
```diff
@@ -0,0 +1,113 @@
+layout: redirect
+---
+
+In this setup the DataHub is deployed into a Kubernetes environment using the Edge operator. The {{< product-c8y-iot >}} DataHub backend is run as a microservice within the {{< product-c8y-iot >}} platform. The Dremio master and executor are deployed as a set of the Kubernetes pods.
```
```suggestion
In this setup the DataHub is deployed into a Kubernetes environment using the Edge operator. The {{< product-c8y-iot >}} DataHub backend is run as a microservice within the {{< product-c8y-iot >}} platform. The Dremio master and executor are deployed as a set of Kubernetes pods.
```
Doesn't quite scan as good english

rthrippleton commented on content/edge-kubernetes/edge-custom-resource-definition-bundle/k8-edge-dremio-cred-secret.md:
```diff
@@ -0,0 +1,14 @@
+|<div style="width:150px">Field</div>|Required|<div style="width:70px">Type</div>|Default|Description|
+|:---|:---|:---|:---|:---|
+|DREMIO_ADMIN_USER|Yes|String||Dremio admin username.
+|DREMIO_ADMIN_PASSWORD|Yes|String||Dremio admin password. <p><p>**Important:** The password must be at least 8 letters long, must contain at least one number and one letter.
```
> The password must be at least 8 letters long, must contain at least one number and one letter.

Out of interest, is this a password requirement that the dremio backend is enforcing on us?

(otherwise, I'd find this quite annoying to enforce password requirements in a low-level API)

bbyreddy replied:
Right, this is the password policy enforced by Dremio and it looks like we don't have a control.

rthrippleton commented on content/edge-kubernetes/edge-custom-resource-definition-bundle/k8-edge-datahub.md:
```diff
@@ -0,0 +1,15 @@
+layout: redirect
+---
+
+Enables the installation and configuration of the {{< product-c8y-iot >}} DataHub Edge. Installing the DataHub requires additional system resources, including at least 10 extra CPU cores and 10 GB of RAM on the Kubernetes node. Additionally, the DataHub will make use of five Persistent Volumes (PVs) for the DataHub MySQL, Dremio master, executor, executor-cloud, and Zookeeper pods. See [Configuri ...
```
Check with the CDH team, but describing a CDH installation as "**the** DataHub [Edge]" feels like it doesn't fit with the normal branding. I'd expect to see "Installing DataHub requires ... "

("the DataHub MySQL, ..., ... pods" is fine though, as 'the' is referring to specific tangible single things)

bbyreddy replied:
DataHub Edge has been used in the earlier versions, hence continued with that. @BeateRixen , what do you suggest?

BeateRixen commented on content/change-logs/edge/edge-on-kubernetes-2025.0.0-datahub.md:
```diff
@@ -0,0 +1,19 @@
+---
+date: 2025-02-07
```
The date should either be left empty for a future deployment or, if this release note should be published with the 2025.0 release today, it must be set to March 31st.

bbyreddy replied:
I marked it as 31st March, though it might not be published today, but should be listed under the 2025.0 release.

BeateRixen commented on content/change-logs/edge/edge-on-kubernetes-2025.0.0-datahub.md:
```diff
@@ -0,0 +1,19 @@
+---
+date: 2025-02-07
+title: DataHub
```
This is a strange title for the ticket?

bbyreddy replied:
I changed it to "Cumulocity DataHub installation in Edge on Kubernetes". Otherwise please suggest an appropriate title :)

BeateRixen commented on content/change-logs/edge/edge-on-kubernetes-2025.0.0-datahub.md:
```diff
@@ -0,0 +1,19 @@
+    label: Edge on Kubernetes
+build_artifact:
+  - value: tc-nJH2U7g3u
+    label: edge-operator
```
Shouldn´t this rather be "datahub"?

bbyreddy replied:
This announcement can also go into DataHub, but here we want to highlight that the DataHub can now be installed via the Edge operator. This wasn't possible earlier as the user was expected to download an archive and follow certain steps to perform the DataHub installation.

BeateRixen commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
```diff
@@ -0,0 +1,113 @@
+#### Resource requirements
+
+The resource requirements for running a bare {{< product-c8y-iot >}} Edge instance are described in [Requirements](/edge-kubernetes/installing-edge-on-k8/#prerequisites).
+When {{< product-c8y-iot >}} DataHub Edge on Kubernetes is deployed on top, the resource requirements change by following additional amount:
```
```suggestion
When {{< product-c8y-iot >}} DataHub Edge on Kubernetes is deployed on top, the resource requirements change by the following additional amounts:
```

BeateRixen commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
```diff
@@ -0,0 +1,113 @@
+### Setting up {{< product-c8y-iot >}} DataHub Edge on Kubernetes
+To install and configure DataHub Edge on Kubernetes, update the `spec.dataHub` field in the Edge Custom Resource (CR) with the necessary configuration details for the Edge operator. After making the changes, apply the updated CR to deploy DataHub Edge.
+
+For more details on the `spec.dataHub` field, refer [Edge Custom Resource - DataHub](/edge-kubernetes/edge-custom-resource-definition/#dataHub).
```
```suggestion
For more details on the `spec.dataHub` field, refer to [Edge Custom Resource - DataHub](/edge-kubernetes/edge-custom-resource-definition/#dataHub).
```

BeateRixen commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
```diff
@@ -0,0 +1,113 @@
+
+{{< product-c8y-iot >}} DataHub Edge on Kubernetes behaves like the Cloud and Edge appliance version.
+
+### Validation of {{< product-c8y-iot >}} DataHub Installation
```
```suggestion
### Validation of {{< product-c8y-iot >}} DataHub installation
```

BeateRixen commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
```diff
@@ -0,0 +1,113 @@
+If the product doesn't work as intended after the installation, go through the validation steps described below.
+
+{{< c8y-admon-info >}}
+Substitute the namespace name *c8yedge* in the subsequent commands with the specific namespace name into which you installed Edge .
```
```suggestion
Substitute the namespace name *c8yedge* in the subsequent commands with the specific namespace name into which you installed Edge.
```

BeateRixen commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
```diff
@@ -0,0 +1,113 @@
+
+#### {{< product-c8y-iot >}} DataHub microservice
+
+When logged into {{< product-c8y-iot >}} UI, the {{< product-c8y-iot >}} DataHub microservice is available under *Administration > Ecosystem > Microservices*.
```
```suggestion
When logged into the {{< product-c8y-iot >}} UI, the {{< product-c8y-iot >}} DataHub microservice is available under **Administration > Ecosystem > Microservices**.
```

BeateRixen commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
```diff
@@ -0,0 +1,113 @@
+
+#### DataHub web application
+
+When logged into the {{< product-c8y-iot >}} UI, the {{< product-c8y-iot >}} DataHub web application is available under *Administration > Ecosystem > Applications*.
```
```suggestion
When logged into the {{< product-c8y-iot >}} UI, the {{< product-c8y-iot >}} DataHub web application is available under **Administration > Ecosystem > Applications**.
```

bbyreddy committed:
Update c8yedge.yaml

bbyreddy committed:
Addressed review comments

bbyreddy committed:
Apply suggestions from code review

Co-authored-by: Richard Thrippleton <richard.thrippleton@softwareag.com>
Co-authored-by: Beate Rixen <90445236+BeateRixen@users.noreply.github.com>

bbyreddy committed:
Merge branch 'edge/y2025-updates-first-cut' into edge/y2025-datahub-integration

bbyreddy committed:
Addressed review comments.

BeateRixen commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
```diff
@@ -0,0 +1,113 @@
+layout: redirect
+---
+
+In this setup the DataHub is deployed into a Kubernetes environment using the Edge operator. The {{< product-c8y-iot >}} DataHub backend is run as a microservice within the {{< product-c8y-iot >}} platform. The Dremio master and executor are deployed as a set of Kubernetes pods.
```
```suggestion
In this setup, {{< product-c8y-iot >}} DataHub is deployed into a Kubernetes environment using the Edge operator. The  DataHub backend is run as a microservice within the {{< product-c8y-iot >}} platform. The Dremio master and executor are deployed as a set of Kubernetes pods.
```

BeateRixen commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
```diff
@@ -0,0 +1,113 @@
+
+{{< product-c8y-iot >}} DataHub Edge on Kubernetes behaves like the Cloud and Edge appliance version.
+
+### Validation of {{< product-c8y-iot >}} DataHub installation
```
```suggestion
### Validation of the {{< product-c8y-iot >}} DataHub installation {#validation-of-the-datahub-edge-installation}
```

BeateRixen commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
```diff
@@ -0,0 +1,113 @@
+
+In this setup the DataHub is deployed into a Kubernetes environment using the Edge operator. The {{< product-c8y-iot >}} DataHub backend is run as a microservice within the {{< product-c8y-iot >}} platform. The Dremio master and executor are deployed as a set of Kubernetes pods.
+
+### Prerequisites
```
```suggestion
### Prerequisites {#prerequisites}
```

BeateRixen commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
```diff
@@ -0,0 +1,113 @@
+
+### Prerequisites
+
+#### Resource requirements
```
```suggestion
#### Resource requirements {#resource requirements}
```

BeateRixen commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
```diff
@@ -0,0 +1,113 @@
+
+ Hardware requirements for the host OS are excluded.
+
+### Setting up {{< product-c8y-iot >}} DataHub Edge on Kubernetes
```
```suggestion
### Setting up {{< product-c8y-iot >}} DataHub Edge on Kubernetes {#setting-up-datahub-edge-on-kubernetes}
```

BeateRixen commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
```diff
@@ -0,0 +1,113 @@
+
+In order to access Dremio, you must also make the domain ``datahub-<domain_name>`` resolvable, just as the configured domain name and ``management-<domain_name>`` were made resolvable in [Accessing Edge](/edge-kubernetes/installing-edge-on-k8/#accessing-edge).
+
+#### Using {{< product-c8y-iot >}} DataHub Edge on Kubernetes
```
```suggestion
#### Using {{< product-c8y-iot >}} DataHub Edge on Kubernetes {#using-datahub-edge-on-kubernetes}
```

BeateRixen commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
```diff
@@ -0,0 +1,113 @@
+Substitute the namespace name *c8yedge* in the subsequent commands with the specific namespace name into which you installed Edge.
+{{< /c8y-admon-info >}}
+
+#### MySQL
```
```suggestion
#### MySQL {#mysql}
```

BeateRixen commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
```diff
@@ -0,0 +1,113 @@
+
+#### MySQL
+
+You can monitor the startup of the MySQL pod ``datahub-mysql-0`` using
```
```suggestion
You can monitor the startup of the MySQL pod ``datahub-mysql-0`` using:
```

BeateRixen commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
````diff
@@ -0,0 +1,113 @@
+mysql-client  ClusterIP     XXX.XXX.XXX.XXX     <none>               3306/TCP         10m
+```
+
+#### Dremio
````
```suggestion
#### Dremio {#dremio}
```

BeateRixen commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
```diff
@@ -0,0 +1,113 @@
+
+#### Dremio
+
+You can monitor the state of the Dremio pods "zk-0", "dremio-executor-0", and "dremio-master-0" using
```
```suggestion
You can monitor the state of the Dremio pods "zk-0", "dremio-executor-0", and "dremio-master-0" using:
```

BeateRixen commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
````diff
@@ -0,0 +1,113 @@
+dremio-client     LoadBalancer      XXX.XXX.XXX.XXX         XXX.XXX.XXX.XXX           31010:XXXXX/TCP,9047:XXXXX/TCP,32010:XXXXX/TCP  9m33s
+```
+
+#### {{< product-c8y-iot >}} DataHub microservice
````
```suggestion
#### {{< product-c8y-iot >}} DataHub microservice {#datahub-microservice}
```

BeateRixen commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
```diff
@@ -0,0 +1,113 @@
+
+When logged into the {{< product-c8y-iot >}} UI, the {{< product-c8y-iot >}} DataHub microservice is available under **Administration > Ecosystem > Microservices**.
+
+You can monitor the startup of the microservice pod "datahub-scope-edge-deployment-...." using
```
```suggestion
You can monitor the startup of the microservice pod "datahub-scope-edge-deployment-...." using:
```

BeateRixen commented on content/datahub/running-datahub-on-the-edge-bundle/setting-up-datahub-edge-on-k8s.md:
````diff
@@ -0,0 +1,113 @@
+c8yedge       datahub-scope-edge-deployment-XXXXXXXXXX-YYYYY  1/1     Running     0           16m
+```
+
+#### DataHub web application
````
```suggestion
#### DataHub web application {#datahub-web-application}
```

BeateRixen commented on content/edge-kubernetes/edge-custom-resource-definition-bundle/k8-edge-datahub.md:
```diff
@@ -0,0 +1,15 @@
+layout: redirect
+---
+
+Specifying this field installs and configures the {{< product-c8y-iot >}} DataHub Edge. Installing the DataHub requires additional system resources, including at least 10 extra CPU cores and 10 GB of RAM on the Kubernetes node. Additionally, the DataHub will make use of five Persistent Volumes (PVs) for the DataHub MySQL, Dremio master, executor, executor-cloud, and Zookeeper pods. See [Configuri ...
```
```suggestion
Specifying this field installs and configures the {{< product-c8y-iot >}} DataHub Edge. Installing DataHub requires additional system resources, including at least 10 extra CPU cores and 10 GB of RAM on the Kubernetes node. Additionally, DataHub will make use of five Persistent Volumes (PVs) for the DataHub MySQL, Dremio master, executor, executor-cloud, and Zookeeper pods. See [Configuring storage](/edge-kubernetes/installing-edge-on-k8/#configuring-storage) for more details about Persistent Volumes.
```

BeateRixen commented on content/edge-kubernetes/edge-custom-resource-definition-bundle/k8-edge-datahub.md:
```diff
@@ -0,0 +1,15 @@
+
+Specifying this field installs and configures the {{< product-c8y-iot >}} DataHub Edge. Installing the DataHub requires additional system resources, including at least 10 extra CPU cores and 10 GB of RAM on the Kubernetes node. Additionally, the DataHub will make use of five Persistent Volumes (PVs) for the DataHub MySQL, Dremio master, executor, executor-cloud, and Zookeeper pods. See [Configuri ...
+
+This field is optional, and if omitted, the DataHub Edge will not be installed.
```
```suggestion
This field is optional, and if omitted, DataHub Edge will not be installed.
```

BeateRixen commented on content/edge-kubernetes/edge-custom-resource-definition-bundle/k8-edge-datahub.md:
```diff
@@ -0,0 +1,15 @@
+
+|<div style="width:150px">Field</div>|Required|<div style="width:115px">Type</div>|Default|Description|
+|:---|:---|:---|:---|:---|
+|enabled|Yes|String||Set to `true` to install the {{< product-c8y-iot >}} DataHub Edge.
```
```suggestion
|enabled|Yes|String||Set to `true` to install {{< product-c8y-iot >}} DataHub Edge.
```

BeateRixen commented on content/edge-kubernetes/edge-custom-resource-definition-bundle/k8-edge-datahub.md:
```diff
@@ -0,0 +1,15 @@
+|:---|:---|:---|:---|:---|
+|enabled|Yes|String||Set to `true` to install the {{< product-c8y-iot >}} DataHub Edge.
+|dremioAdminCredentialsSecretName|Yes|String||Name of the Kubernetes Secret containing the credentials for admin access to Dremio. For more information, see [Dremio Admin Credentials Secret](/edge-kubernetes/edge-custom-resource-definition/#k8-edge-dremio-cred-secret). <p><p>**Info:** The Edge operator retrieves this secret from the **`EDGE-CR-NAMESPACE`**. Ensure that this secret is created befo ...
+|resources.limits|No|Structure|Defaults to CPU Limit: 2<br>Memory Limit: 4096Mi|Specify resource limits for the Dremio master and executor pod. <p><p>**Important:** CPU must be specified as an integer (number of CPUs) without the units, for example, 2 for 2 CPUs instead of 2000m or 2M and Memory must be specified in Mi without the units, for example 4096 instead of 4096Mi or 4Gi. <p><p>For more i ...
```
```suggestion
|resources.limits|No|Structure|Defaults to CPU limit: 2<br>Memory limit: 4096Mi|Specify resource limits for the Dremio master and executor pod. <p><p>**Important:** CPU must be specified as an integer (number of CPUs) without the units, for example, 2 for 2 CPUs instead of 2000m or 2M and Memory must be specified in Mi without the units, for example 4096 instead of 4096Mi or 4Gi. <p><p>For more information, see [Resource limits specification](/edge-kubernetes/edge-custom-resource-definition/#k8-edge-resources-limits-spec).
```

BeateRixen commented on content/edge-kubernetes/edge-custom-resource-definition-bundle/k8-edge-dremio-cred-secret.md:
```diff
@@ -0,0 +1,14 @@
+layout: redirect
+---
+
+Specifies the Kubernetes secret containing the credentials for admin access to Dremio. The Edge operator retrieves this secret from the namespace **`EDGE-CR-NAMESPACE`**. It is important that this secret is created before initiating the Edge deployment or update process.
```
**`EDGE-CR-NAMESPACE`**
Either format in bold or as code. Double-highlighting is not required.

bbyreddy replied:
Addressed in the https://github.com/Cumulocity-IoT/c8y-docs/pull/3131/commits/08ee91d5f5030af3b6fd6d0c1481d7ddd69b630d

BeateRixen commented on content/edge-kubernetes/edge-custom-resource-definition-bundle/k8-edge-specification.md:
```diff
@@ -13,10 +13,11 @@ This section defines the Edge deployment's configurations.
 |mongodb|No|Structure||Configurations needed to deploy the MongoDB server managed by the Edge operator or connect to an external one. For more information, see [MongoDB](/edge-kubernetes/edge-custom-resource-definition/#k8-edge-mongodb).
-|microservices|No|Array of Structure|The Edge operator deploys all the default {{< product-c8y-iot >}} microservices, which include the Apama, Smart Rules, OPCUA Management Server microservices.|Specify resources to allocate to each of the default {{< product-c8y-iot >}} microservices deployed. For more information, see [Microservices](/edge-kubernetes/edge-custom-resource-definition/#k8-edge-mic ...
+|microservices|No|Array of Structure|The Edge operator deploys all the default {{< product-c8y-iot >}} microservices, which include the Apama, Smart Rules, OPCUA Management Server microservices.|Specify resources to allocate to each of the default {{< product-c8y-iot >}} microservices deployed. For more information, see [Microservices](/edge-kubernetes/edge-custom-resource-definition/#k8-edge-mic ...
+|dataHub|No|Structure|{{< product-c8y-iot >}} DataHub is not installed if omitted.|Specifying this field installs and configures the {{< product-c8y-iot >}} DataHub Edge. For more information, see [DataHub](/edge-kubernetes/edge-custom-resource-definition/#k8-edge-datahub).
```
```suggestion
|dataHub|No|Structure|{{< product-c8y-iot >}} DataHub is not installed if omitted.|Specifying this field installs and configures {{< product-c8y-iot >}} DataHub Edge. For more information, see [DataHub](/edge-kubernetes/edge-custom-resource-definition/#k8-edge-datahub).
```

BeateRixen commented on content/edge-kubernetes/installing-edge-on-k8-bundle/configuring-storage.md:
```diff
@@ -37,28 +37,25 @@ The Edge operator requests three PVCs, as outlined in the table below. Each of t
-|2 GB|`pulsar-zookeeper-data-pulsar-zookeeper-0`|Claimed by the Pulsar Zookeeper pod, which is deployed only when the Messaging Service is enabled.
+|<div style="width:150px">Persistent Volume</div>|<div style="width:400px">Persistent Volume Claim</div>|<div style="width:500px">Description</div>|
+|:---|:---|:---|
+|75 GB|`mongod-data-edge-db-rs0-0`|Claimed by the MongoDB server to retain application data. The default size is 75 GB, but this value can be adjusted using the `spec.mongodb.resources.requests.storage` field in the Edge CR file. For more detail, see [Edge Custom Resource - MongoDB](/edge-kubernetes/edge-custom-resource-definition/#k8-edge-mongodb)|
```
```suggestion
|75 GB|`mongod-data-edge-db-rs0-0`|Claimed by the MongoDB server to retain application data. The default size is 75 GB, but this value can be adjusted using the `spec.mongodb.resources.requests.storage` field in the Edge CR file. For more details, see [Edge Custom Resource - MongoDB](/edge-kubernetes/edge-custom-resource-definition/#k8-edge-mongodb).|
```

bbyreddy committed:
Apply suggestions from code review

Co-authored-by: Beate Rixen <90445236+BeateRixen@users.noreply.github.com>

bbyreddy committed:
Removed double-highlighting of the EDGE-CR-NAMESPACE

bbyreddy committed:
Addressed review comments

rthrippleton committed:
Remove all doc related to Edge appliance + Datahub

Edge appliance 10.18 + Datahub 2025 is no longer a valid combination
However, we still have the Edge appliance 10.18 doc in here for Reasons,
so we're "vendoring" the old Datahub+Edge doc in here.

rthrippleton committed:
Accidentally pulled in an ancient copy of Datahub-on-Edge-appliance doc, taking it from release/y2025

rthrippleton committed:
Update the sector tag; left it in when copying from DataHub doc

rthrippleton committed:
Maybe needs to be a unique weight?

rthrippleton committed:
Removing the redundant datahub section under edge/

bbyreddy committed:
Merge branch 'edge/y2025-updates-first-cut' into edge/y2025-datahub-integration

bbyreddy committed:
Update c8yedge.yaml

bbyreddy committed:
Merge branch 'edge/y2025-updates-first-cut' into edge/y2025-datahub-integration

bbyreddy committed:
Merge branch 'edge/y2025-updates-first-cut' into edge/y2025-datahub-integration
